ui/components/MeshModelRegistry/MesheryTreeView.js

```diff
@@ -465,22 +469,28 @@ const MesheryTreeView = ({
       <div>
         {width < 1370 && isSearchExpanded ? null : (
           <div style={{ display: 'flex', flexDirection: 'row' }}>
-            <Tooltip title="Expand All" placement="top">
-              <IconButton onClick={expandAll} size="large" disableRipple>
-                <KeyboardArrowDownIcon />
+            <CustomTextTooltip title="Expand All" placement="top" backgroundColor="#3C494F">
+              <IconButton
+                onClick={expandAll}
+                size="large"
+                disableRipple
+                disabled={disabledExpand()}
+              >
+                <ExpandAllIcon height={15} width={15} />
               </IconButton>
-            </Tooltip>
+            </CustomTextTooltip>
 
-            <Tooltip title="Collapse All" placement="top">
+            <CustomTextTooltip title="Collapse All" placement="top" backgroundColor="#3C494F">
```

Comment:
Use the theme.

Reply:
Thankyou for pointing out, I was testing color and forget to update with theme before pushing